perception/sub8_pointcloud/launch/ogrid.launch

```diff
@@ -2,7 +2,7 @@
     <node pkg="sub8_pointcloud" type="ogrid_generator" name="ogrid_pointcloud" >
         <rosparam>
             resolution: 0.2
-            ogrid_size: 50.0
+            ogrid_size: 150
```

Comment:
Parameters like this should have a comment describing what they are and what unit's they're in.